src/Sulu/Bundle/SecurityBundle/Command/CreateUserCommand.php

```diff
@@ -143,7 +96,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
             return 1;
         }
 
-        if (!\in_array($locale, $this->locales)) {
+        if (!\in_array($locale, $this->locales, true)) {
```

Comment:
In order to avoid any uninteded side effects, I would prefer to keep changes to a minimum, so I would not change this in this PR.